xml/Microsoft.Extensions.Primitives/StringValues.xml

```diff
@@ -975,10 +996,18 @@
         <Parameter Name="values" Type="Microsoft.Extensions.Primitives.StringValues" />
       </Parameters>
       <Docs>
-        <param name="values">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="values">A <see cref="T:Microsoft.Extensions.Primitives.StringValues" /> to implicitly convert.</param>
+        <summary>Defines an implicit conversion of a given <see cref="T:Microsoft.Extensions.Primitives.StringValues" /> to a string, with multiple values joined as a comma separated string.</summary>
         <returns>To be added.</returns>
```

Comment:
```suggestion
        <returns>Returns <see langword="null" /> where <see cref="T:Microsoft.Extensions.Primitives.StringValues" /> has been initialized from an empty string array or is <see cref="P:Microsoft.Extensions.Primitives.StringValues.Empty" />. Otherwise, it returns the implicit conversion of <see cref="T:Microsoft.Extensions.Primitives.StringValues" /> to a string.
</returns>
```